docs/content/examples/tutorials/dovecot-solr.md

```diff
@@ -24,36 +24,36 @@ As the official DMS image does not provide `dovecot-solr`, you'll need to includ
 !!! quote ""
 
     === "`user-patches.sh`"
-    
+
         If you'd prefer to avoid a custom image build. This approach is simpler but with the caveat that any time the container is restarted, you'll have a delay as the package is installed each time.
-    
+
         ```bash
         #!/bin/bash
-    
+
         apt-get update && apt-get install dovecot-solr
         ```
-    
+
     === "`compose.yaml`"
-    
+
         A custom DMS image does not add much friction. You do not need a separate `Dockerfile` as Docker Compose supports building from an inline `Dockerfile` in your `compose.yaml`.
-    
+
         The `image` key of the service is swapped for the `build` key instead, as shown below:
-    
+
         ```yaml
         services:
           mailserver:
             hostname: mail.example.com
             # The `image` setting now represents the tag for the local build configured below:
-            image: local/dms:14.0
+            image: local/dms:latest
```

Comment:
I'm ok with this, although `:latest` is generally an anti-pattern, so the idea was to encourage version pinning.

I suppose a CLI command that forces a build without cache may work for a user wanting to upgrade, but as per semver this is a bad practice for major releases 😓 

I think this is valid syntax to make it dynamic, yet explicit, with a fallback:

```suggestion
            image: local/dms:${DMS_TAG:-14.0}
```

This [is also valid](https://docs.docker.com/compose/how-tos/environment-variables/variable-interpolation/#interpolation-syntax):

```suggestion
            image: local/dms:${DMS_TAG?Must set DMS image tag}
```

```bash
docker compose -e DMS_TAG='14.0' up -d
```